### ISSUE TITLE
minor ui bug with system update

### ISSUE THREAD

**Issue by antonkulaga** [rkimoakbioinformatics/oakvar#13]
When I do system update it forgets to close "system update in progress" after finishing the update (at least according to logs). 
Opening oakvar in a new window fixes the issue, so it is very minor bug, but I mention it just in case[](url
![Screenshot from 2022-08-03 20-04-21](https://user-images.githubusercontent.com/842436/182667649-adde4609-63fe-4ba1-b2a2-0862a97597ac.png)
)
In my case it was caused by updating vcf-converter
```
OakVar Server is served at 0.0.0.0:8080
(To quit: Press Ctrl-C or Ctrl-Break)
[2022:08:03 20:02:01] Starting to install vcf-converter:...
[2022:08:03 20:02:06] Downloading code archive of vcf-converter:...
[2022:08:03 20:02:09] Extracting code archive of vcf-converter:...
[2022:08:03 20:02:09] Finished installation of vcf-converter:
```
The update was successful

**Comment by rkimoakbioinformatics** [comment]
@antonkulaga Thanks. I'll look into it. 

By the way, I appreciate all the issues you and Alex have been raising. Most of the issues you observed are the side effect of some fundamental changes OakVar has made. It may show some rough spots now, but eventually it'll have much stronger foundation soon.

**Comment by rkimoakbioinformatics** [comment]
@antonkulaga The issue has been fixed in v2.5.14.

**Comment by antonkulaga** [comment]
I do not have this exact bug, however when oakvar or modules update I do not see notifications in the ui

**Comment by rkimoakbioinformatics** [comment]
@antonkulaga Just released v2.5.22. The UI about module updates and system module update on the GUI has been updated.
